messaging/WFLY-6684_jms_bridge_statistics.adoc

```diff
@@ -0,0 +1,62 @@
+= JMS Bridge should display statistics about messages that have been processed.
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+An Artemis JMS Bridge consumes messages from a source JMS destination, and send them to a target JMS destination.
+Typically either the source or the target destinations are on different servers. The bridge can also be used to bridge messages from other non Artemis
+JMS servers, as long as they are JMS 1.1 compliant.
+The goal of this RFE is to display statistics on the number of processed messages and how many were aborted/rolled back.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-6684
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/ENTMQBR-1004
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:mnovak@redhat.com[Miroslav Novak]
+
+=== Affected Projects or Components
+
+* Wildfly
+* ActiveMQ Artemis (finished)
+
+=== Other Interested Projects
+
+== Requirements
+
+* The messageCount and abortedMessageCount attributes of ActiveMQ Artemis JMS Bridge configuration must be readable via Wildfly management model.
+
+== Implementation Plan
+
+The `message-count` and `aborted-message-count` attributes will be added to the jms bridge resource.
```

Comment:
`message-count` is pretty clear to me. I however don't know how messages in `aborted-message-count` will be count (under which circumstances the message is added here)? Could you please help me?

Also, please add their type and if they are just read-only (I guess they are).

Reply:
fixed